sktime/transformations/series/compose.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+from sktime.transformations.base import _SeriesToSeriesTransformer
+from sktime.utils.validation.series import check_series
+
+from sklearn.base import clone
+from sklearn.utils.metaestimators import if_delegate_has_method
+
+
+class OptionalPassthrough(_SeriesToSeriesTransformer):
+    """A transformer to tune the implicite hyperparameter whether to use a
+    particular transformer inside a pipeline (e.g. TranformedTargetForecaster)
+    or not.This is achived by having the additional hyperparameter
+    \"passthrough\" which can be added to a grid then (see example).
+
+    Parameters
+    ----------
+    transformer : Estimator
+        scikit-learn-like or sktime-like transformer to fit and apply to series
+    passthrough : bool
+        This arg decides whether to apply the given transformer or to just
+        passthrough the data (identity transformation)
+
+    Examples
+    ----------
+    >>> from sktime.datasets import load_airline
+    >>> from sktime.forecasting.naive import NaiveForecaster
+    >>> from sktime.transformations.series.compose import OptionalPassthrough
+    >>> from sktime.transformations.series.boxcox import BoxCoxTransformer
+    >>> from sktime.transformations.series.detrend import Deseasonalizer
+    >>> from sktime.forecasting.compose import TransformedTargetForecaster
+    >>> from sktime.forecasting.model_selection import (
+        ForecastingGridSearchCV,
```

Comment:
I think we have to add `...` for continued lines

Reply:
@mloning  where exactly? I used a docstring example also [here ](https://www.sktime.org/en/latest/api_reference/modules/auto_generated/sktime.forecasting.model_evaluation.evaluate.html) without `...` but seems the indentation is not perfectly. I couldnt find any reference of how to exactly write such example code snippets with Numpy docstring style, do you have any reference here?

Reply:
maybe we better remove `>>> ` from it to make the code copy-pasteable on the website?

Reply:
The `>>>` is necessary, I also couldn't find any style guide, so trail and error, it's very similar to the IPython command line, Pycharm also seemed to highlight bad code snippets.

Reply:
I think you add this to the website to toggle them on and off - this is what scikit-learn does: check out what happens when you hover over the examples section: https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html